Packs/CommonPlaybooks/pack_metadata.json

```diff
@@ -1,14 +1,292 @@
-{
-    "name": "Common Playbooks",
-    "description": "Frequently used playbooks pack.",
-    "support": "xsoar",
-    "currentVersion": "1.0.1",
-    "author": "Cortex XSOAR",
-    "url": "https://www.paloaltonetworks.com/cortex",
-    "email": "",
-    "created": "2020-05-14T08:33:25Z",
-    "categories": [],
-    "tags": [],
-    "useCases": [],
-    "keywords": []
+{
+    "name": "Common Playbooks",
+    "description": "Frequently used playbooks pack.",
+    "support": "xsoar",
+    "currentVersion": "1.0.1",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "created": "2020-05-14T08:33:25Z",
+    "categories": [],
+    "tags": [],
+    "useCases": [],
+    "keywords": [],
+    "dependencies": {
+        "VirusTotal": {
+            "mandatory": false,
+            "display_name": "VirusTotal"
+        },
+        "ExtraHop": {
+            "mandatory": false,
+            "display_name": "ExtraHop Reveal(x)"
+        },
+        "CommonScripts": {
+            "mandatory": true,
+            "display_name": "Common Scripts"
+        },
+        "JoeSecurity": {
+            "mandatory": false,
+            "display_name": "Joe Security"
+        },
+        "SNDBOX": {
+            "mandatory": false,
+            "display_name": "SNDBOX"
+        },
+        "McAfee-TIE": {
+            "mandatory": false,
+            "display_name": "McAfee Threat Intelligence Exchange"
+        },
+        "GmailSingleUser": {
+            "mandatory": false,
+            "display_name": "Gmail Single User (Beta)"
+        },
+        "Flashpoint": {
+            "mandatory": false,
+            "display_name": "Flashpoint"
+        },
+        "CarbonBlackProtect": {
+            "mandatory": false,
+            "display_name": "Carbon Black Enterprise Protection"
+        },
+        "CalculateTimeDifference": {

```

Comment:
@idovandijk you have deleted `CalculateTimeDifference` pack. So you should remove it as dependency also.